iot-e2e-tests/common/src/main/java/com/microsoft/azure/sdk/iot/common/tests/iothub/serviceclient/ServiceClientTests.java

```diff
@@ -75,9 +85,42 @@ public static Collection inputsCommon() throws IOException
         return inputs;
     }
 
+    @BeforeClass
+    public static void startProxy()
+    {
+        proxyServer = DefaultHttpProxyServer.bootstrap()
+                .withPort(testProxyPort)
```

Comment:
are 8 spaces of indentation normal for Java?

Reply:
Not quite, I can fix this

Reply:
What is normal?

Reply:
One tab over from the method that started the fluent chain:

Although github won't let me demonstrate exactly how this looks, the latest view of this file should show you.